src/lib/operators/aggregate.cpp

```diff
@@ -123,6 +123,7 @@ template <typename ColumnDataType, typename AggregateType, typename AggregateKey
 struct AggregateContext : public AggregateResultContext<ColumnDataType, AggregateType> {
   AggregateContext() {
     auto allocator = AggregateResultIdMapAllocator<AggregateKey>{&this->buffer};
+    // NOLINTNEXTLINE(clang-analyzer-core.CallAndMessage) - false warning: called C++ object (result_ids) is null
```

Comment:
Maybe add `: result_ids(nullptr)` instead?

Reply:
This does not resolve the warning.
`: result_ids{std::make_unique<AggregateResultIdMap<AggregateKey>>()}`
or
`: result_ids{nullptr}`
still result in a warning:
````
/hyrise/src/lib/operators/aggregate.cpp:127:5: error: Called C++ object pointer is null [clang-analyzer-core.CallAndMessage,-warnings-as-errors]
    result_ids = std::make_unique<AggregateResultIdMap<AggregateKey>>(allocator);
    ^
````

Reply:
Ok, I don't understand its problem.